src/slack/slack.js

```diff
@@ -765,22 +688,21 @@ export class Slack {
 
       // home-personal-urgent
       if (order.isHome) {
-        await submitOrder(MESSAGES.home.personal, true)
-        return true
+        order.messages = MESSAGES.home.personal
       }
 
       // office-personal-urgent
-      await updateMessage({
+      await this.updateMessage(order, {
         actions: ORDER_NOTE_ACTIONS,
         fields: [...this.getOrderFields(order), {title: ':pencil: Do you want to add a note to the order?'}],
       })
-      return false
     }
 
     // office-personal-?-note
     if (actionName === 'note') {
-      await submitOrder(actionValue === 'note-yes' ? MESSAGES.office.personal.note : null)
-      return true
+      if (actionValue === 'note-yes') {
+        order.messages = MESSAGES.office.personal.note
+      }
```

Comment:
what happens when `note-no` is picked? have you tested the scenario? I'd guess we need to call the submit to the db from here, because no further user action is required. am I right?

Reply:
you are right. I added submitOrder() and put final (DB, notify_manager) code there, so we can call it from `handleUserMessage()` and `handleUserActions()`